src/main/scala/io/iohk/ethereum/network/package.scala

```diff
@@ -19,4 +25,38 @@ package object network {
       Hex.decode(nodeId)
     curve.getCurve.decodePoint(bytes)
   }
+
+  object AsymmetricCipherKeyPairSerializable {
+
+    def toBytes(keysValuePair: AsymmetricCipherKeyPair): (String, String) = {
+      val pubKey = keysValuePair.getPublic.asInstanceOf[ECPublicKeyParameters].toNodeId
+      val privKey = keysValuePair.getPrivate.asInstanceOf[ECPrivateKeyParameters].getD.toByteArray
+      (Hex.toHexString(pubKey), Hex.toHexString(privKey))
+    }
+
+    def fromBytes(publicKey: String, privateKey: String): AsymmetricCipherKeyPair = {
+      val pub = new ECPublicKeyParameters(publicKeyFromNodeId(publicKey), curve)
+      val priv = new ECPrivateKeyParameters(new BigInteger(Hex.decode(privateKey)), curve)
+      new AsymmetricCipherKeyPair(pub, priv)
+    }
+
+  }
+
+  def loadAsymmetricCipherKeyPair(filePath: String): AsymmetricCipherKeyPair = {
+    if(!new File(filePath).exists()){
+      val keysValuePair = generateKeyPair()
+
+      //Write keys to file
+      val (pub, priv) = AsymmetricCipherKeyPairSerializable.toBytes(keysValuePair)
+      val file = new PrintWriter(filePath)
+      file.write(pub ++ "\n" ++ priv)
```

Comment:
@adamsmo me too but I guess this task was only for storing this in plain text. That being said +1 to address this in the future as @mcsherrylabs said :smile: